starter-code/pizza.js

```diff
@@ -1 +1,95 @@
 // Write your Pizza Builder JavaScript in this file.
+
+$(document).ready(function() {
+  $('.green-pepper').toggle();
```

Comment:
I noticed that you guys went into the css and altered some of the classes to force some of the behavior that you guys wanted, namely to hide things like the list items and ingredients upon page load. 

What you guys did is definitely valid however there are some other tools you coulf have used. Something like the following could have achieved the same affect using just javascript:

  //Sauce is pre-initialized to normal red sauce;
 $('.sauce').removeClass('sauce-white');
 //Crust is pre-initialized to regular crust
 $('.crust').removeClass('crust-gluten-free');

//Removes the class active from all buttons.
$('.active').removeClass('active');

//Hides the pepperoni elements from the pizza.
$('.pep').hide();

//Hides the mushroom elements from the pizza.
$('.mushroom').hide();

//Hides the green pepper elements from the pizza.
$('.green-pepper').hide();

//Hides the extra ingredients from the price list.
$('.price > ul').children().hide();